code/drasil-example/Drasil/DblPendulum/Body.hs

```diff
@@ -29,7 +29,7 @@ import Drasil.DblPendulum.Figures (figMotion)
 import Data.Drasil.Concepts.Math (mathcon, cartesian)
 import Data.Drasil.Quantities.Math (unitVect, unitVectj)
 import Drasil.DblPendulum.Assumptions (assumptions)
-import Drasil.DblPendulum.Concepts (pendulumTitle)
+import Drasil.DblPendulum.Concepts ()
```

Comment:
Since no imports are needed from this file, this line should be entirely deleted.

Reply:
Ah okay, sorry about that. Next time I'll make sure the local make works first. I'll also remove the unneeded import line.